src/server/controllers/LoginController.ts

```diff
@@ -60,18 +64,22 @@ export class LoginController implements LoginControllerInterface {
       const user = await this.authService.verifyOtp(email, otp)
       req.session!.user = user
       res.ok(jsonMessage('OTP hash verification ok.'))
+      return
     } catch (error) {
       if (error instanceof InvalidOtpError) {
         res.unauthorized(
           jsonMessage(
             `OTP hash verification failed, ${error.retries} attempt(s) remaining.`,
           ),
         )
-      } else if (error instanceof NotFoundError) {
+        return
+      }
+      if (error instanceof NotFoundError) {
         res.unauthorized(jsonMessage('OTP expired/not found.'))
-      } else {
-        res.serverError(jsonMessage(error.message))
+        return
       }
+      res.serverError(jsonMessage(error.message))
+      return
```

Comment:
I think we should retain the `else-if` and `else` blocks here for greater code clarity.